tests/kuttl/tests/keystone_scale/03-assert.yaml

```diff
@@ -0,0 +1,63 @@
+#
+# Check for:
+#
+# - 1 KeystoneAPI CR
+# - 3 Pods for KeystoneAPI CR
+#
+
+apiVersion: keystone.openstack.org/v1beta1
+kind: KeystoneAPI
+metadata:
+  finalizers:
+  - KeystoneAPI
+  name: keystone
+  namespace: openstack
+spec:
+  adminProject: admin
+  adminRole: admin
+  adminUser: admin
+  containerImage: quay.io/tripleozedcentos9/openstack-keystone:current-tripleo
+  customServiceConfig: |
+    [DEFAULT]
+    debug = true
+  databaseInstance: openstack
+  databaseUser: keystone
+  debug:
+    bootstrap: false
+    dbSync: false
+    service: false
+  passwordSelectors:
+    admin: AdminPassword
+    database: KeystoneDatabasePassword
+  preserveJobs: false
+  region: regionOne
+  replicas: 3
+  resources:
+    requests:
+      cpu: "1.0"
+      memory: 500Mi
+  secret: osp-secret
+status:
+  apiEndpoint:
+    admin: http://keystone-admin-openstack.apps-crc.testing
+    internal: http://keystone-internal-openstack.apps-crc.testing
+    public: http://keystone-public-openstack.apps-crc.testing
+  databaseHostname: openstack
+  readyCount: 3
+---
+# We can't provide 3 separate Pod YAMLs to match against because we don't know
+# what random strings were used in the Pod names for the KeystoneAPI, so instead
+# we run a bash script to check for the 3 Pods
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+namespaced: true
+commands:
+  - script: |
+      set -x
+      sleep 3
+      podcount=$(oc get pods -n openstack -l service=keystone -o name | wc -l)
+      if [ "$podcount" == "3" ]; then
```

Comment:
This fails to me with:

```
logger.go:42: 15:23:26 | keystone_scale/3-scale-keystoneapi | keystoneapi.keystone.openstack.org/keystone patched
I0105 15:23:27.246030  427800 request.go:655] Throttling request took 1.049253403s, request: GET:https://api.crc.testing:6443/apis/flowcontrol.apiserver.k8s.io/v1beta2?timeout=32s
    logger.go:42: 15:23:29 | keystone_scale/3-scale-keystoneapi | running command: [sh -c set -x
        sleep 3
        podcount=$(oc get pods -n openstack -l service=keystone -o name | wc -l)
        if [ "$podcount" == "3" ]; then
          exit 0
        else
          exit 1
        fi
        ]
    logger.go:42: 15:23:29 | keystone_scale/3-scale-keystoneapi | + sleep 3
    logger.go:42: 15:23:32 | keystone_scale/3-scale-keystoneapi | + oc get pods -n openstack -l service=keystone -o name
    logger.go:42: 15:23:32 | keystone_scale/3-scale-keystoneapi | + wc -l
    logger.go:42: 15:23:32 | keystone_scale/3-scale-keystoneapi | + podcount=3
    logger.go:42: 15:23:32 | keystone_scale/3-scale-keystoneapi | + [ 3 == 3 ]
    logger.go:42: 15:23:32 | keystone_scale/3-scale-keystoneapi | sh: 4: [: 3: unexpected operator
    logger.go:42: 15:23:32 | keystone_scale/3-scale-keystoneapi | + exit 1
```

Reply:
Following the suggestion from @stuggi, I've changed this code to check the number of pods from the deployment, so no more bash here and the error should be gone.